.travis.yml

```diff
@@ -41,6 +41,8 @@ env:
   - VERSION="master" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="89" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
   - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="82" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
   - VERSION="" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="27" TRAVIS_PULL_REQUEST="false"  # To check VERSION empty or unset case.
+  - VERSION="" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="15" TRAVIS_PULL_REQUEST="false" EXCLUDE_LINT="$HOME/build/$TRAVIS_REPO_SLUG/broken_lint"  # To check EXCLUDE_LINT
+  - VERSION="" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="27" TRAVIS_PULL_REQUEST="false" INCLUDE_LINT="$HOME/build/$TRAVIS_REPO_SLUG"  # To check INCLUDE_LINT
```

Comment:
@moylop260 
I add `INCLUDE="test_module"` into .travis.yml 
This is what you meant?